internal/driver/constants.go

```diff
@@ -29,6 +29,9 @@ const (
 	FrameRateValueDenominator       = "FrameRateValueDenominator"
 	FrameRateValueNumerator         = "FrameRateValueNumerator"
 	PathIndex                       = "PathIndex"
+	PixFmtWidth                     = "Width"
+	PixFmtHeight                    = "Height"
```

Comment:
Do these need to be prefixed, or can they just be `Width = "Width"` and `Height = "Height"`?